LINK/usr/bin/appliance_console_cli

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env ruby
 
 require "pathname"
-# NOTE: in dev, running in cfme/system/LINK/bin/appliance_console_cli
-ROOT ||= [
-  "/var/www/miq",
-  File.expand_path(File.join(File.dirname(__FILE__), "../../.."))
-].detect { |f| File.exist?(f) }
-RAILS_ROOT ||= Pathname.new("#{ROOT}/vmdb")
 
-ENV['BUNDLE_GEMFILE'] ||= "#{ROOT}/vmdb/Gemfile"
+# Handle development environment by setting up Gemfile and manageiq location
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../../../manageiq/Gemfile', __dir__)
```

Comment:
I had setup the `ROOT` first to detect rails root across invoking via a symbolic, relative, or absolute path. It also worked if we decided to copy this into `/usr/bin/`.

Having said that, if you are content with this change, then looks good to me.